template/prod.yml

```diff
@@ -86,6 +86,7 @@ services:
         volumes:
             - ./template/keys-generated/certs/storage-proxy/cert.crt:/var/www/storage-proxy/cert/cert.crt:ro
             - ./template/keys-generated/certs/storage-proxy/private.key:/var/www/storage-proxy/cert/private.key:ro
+            - ./storage-proxy/nginx.conf:/usr/local/openresty/nginx/conf/nginx.conf
```

Comment:
just as a side note: this seems to be covered by https://github.com/mendersoftware/integration/blob/14c12f7aeb91ffba397a11b571515f91c75f6303/docker-compose.storage.minio.yml#L28 - but the change here should not interfere with the download links/ uploads anyway, as the proxy doesn't seem to be used in the S3 scenario.

Reply:
@mzedel I was actually meaning to remove the mount from minio.yml - fixed now.

the reason is that when there are competing mounts - say in demo and minio, or more impornantly prod and minio - which one will be selected depends on the order of files in the docker-compose statement...

now - minio has no mount, and it's up to demo/prod/testing to mount one

Reply:
sounds good, the prod and demo distinction makes it more explicit than hidden in the minio mount